.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,25 @@
+name: PR Title
+on:
+  pull_request:
+    branches:
+      - "*"
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: aslafy-z/conventional-pr-title-action@v2.2.5
+        # continue-on-error: true
+        id: title-check
+        with:
+          success-state: PR title is valid
+          failure-state: "PR title does not follow the conventional commit spec: https://www.conventionalcommits.org/en/v1.0.0/"
+          context-name: pr-title
+          preset: conventional-changelog-angular@latest
```

Comment:
`feature` is not allowed? Or you mean it's `feat`? For easier adoption of these rules, I think sticking with a good known one instead of making up our own is probably better.